utils/rjson.hh

```diff
@@ -145,6 +151,19 @@ inline rjson::value empty_string() {
 // The representation is dense - without any redundant indentation.
 std::string print(const rjson::value& value, size_t max_nested_level = default_max_nested_level);
 
+// Writes the JSON value to the output_stream, similar to print() -> string, but
+// directly. Note this has potentially more data copy overhead and should be 
+// reserved for larger values where not creating huge linear strings is useful.
+// (Because data will first be written to N buffers before actually committed to stream, 
+//  where N = <n bytes text>/k (k dependent on buffer/write block behaviour of stream),
+//  and also copied at least 2n times (assuming underlying stream is buffered).
```

Comment:
copied **2** times, not **2n**?

Reply:
copied 2 times, but 2n bytes (or more) copied.

Reply:
Right, but you wrote 2n times, not 2n bytes. Not a biggie, but since I'm testing this patch now, you still have time to send a fixed version if you want.

Reply:
Sure. Fixed.

Reply:
Oops, I accidentally committed the previous version :-( Sorry.